addons/attach/CfgVehicles.hpp

```diff
@@ -86,6 +86,10 @@ class CfgVehicles {
             dayLight = 0;
             onlyInNvg = 1;
             useFlare = 0;
+            maxLifetime = 8 * 60 * 60;
+            blinkingPattern[] = {0.1, 0.9}; // 0.1 s on, 0.9 s off
+            blinkingStartsOn = 1;
+            blinkingPatternGuarantee = 1;
```

Comment:
This property doesn't seem to work currently, you get no blinking at all if you use an on time less than a frame's duration (@dedmen?). But it doesn't hurt adding it.

Reply:
Dunno, should be same as for that other blinking thing :D